frontend/src/components/NodeChart.vue

```diff
@@ -130,8 +109,9 @@ onMounted(async () => {
 const setDefaults = () => {
   // sets page elements back to their default values.
 
-  // set statistics switch to off
-  showStatistics.value = false
+  // set statistics switch to off in the charts store
+  showStatistics.value = false // TODO: remove this line
```

Comment:
TODO here...?

Reply:
I removed the TODO. At one point I wasn't sure if this was needed any longer, but I think it's best to leave it for now.